cocos/2d/CCDrawNode.cpp

```diff
@@ -119,6 +119,7 @@ DrawNode::DrawNode(GLfloat lineWidth)
 , _dirty(false)
 , _dirtyGLPoint(false)
 , _dirtyGLLine(false)
+, _isolated(false)
 , _lineWidth(lineWidth)
 , _defaultLineWidth(lineWidth)
```

Comment:
I think can change to initialize parameters in class declaration.  We should change to use this style when modify some codes.